teb_local_planner/launch/teb_params.yaml

```diff
@@ -51,43 +51,43 @@ bt_navigator:
     use_sim_time: True
     bt_xml_filename: "bt_navigator.xml"
 
-dwb_controller:
+bt_navigator_rclcpp_node:
```

Comment:
Shouldn't think `yaml` file only contain the controller_server code?

Reply:
Is it possible to overwrite only for `controller_server` part?

Reply:
Well in practice all the stuff in that one massive nav2_params yaml shouldn't be done like that, they should be separate yaml files for each package. But yes you can include multiple yaml files, now how it deals with conflicts is unknown to me